### PR TITLE
Feature/app 4840 change occupation details

### PR DESCRIPTION
**Pull request by prabhat-inss**

### CODE REVIEW

File: INSS.EIIR.DataAccess/DatabaseScripts/getEiirExtractXml.sql [hunk 1]
```diff
@@ -276,7 +276,15 @@ CREATE TABLE #Temp
     END AS individualGender,
         
     ISNULL(NULLIF(CONVERT(CHAR(30), individual.date_of_birth, 103), ''), 'No Date of Birth Found') AS individualDOB, 
-    ISNULL(NULLIF(individual.job_title, ''), 'No Occupation Found') AS individualOccupation,
+	CASE 
+	  WHEN individual.job_title IS NULL OR individual.job_title = ''
+		  THEN 'No Occupation Found'
+	  WHEN CHARINDEX('-', individual.job_title, 1) > 1
+		  THEN (SELECT trim(value) FROM STRING_SPLIT(individual.job_title, '-')  ORDER BY (SELECT NULL) OFFSET 1 ROWS FETCH NEXT 1 ROWS ONLY )
```
Comment (VasInsolvencyService) [inline INSS.EIIR.DataAccess/DatabaseScripts/getEiirExtractXml.sql:283]: why do we need FETCH NEXT 1 ROWS ONLY ?

File: INSS.EIIR.DataAccess/DatabaseScripts/getEiirIndex.sql [hunk 2]
```diff
@@ -210,7 +210,15 @@ FROM #Cases c
     END AS individualGender,
         
     ISNULL(NULLIF(CONVERT(CHAR(30), individual.date_of_birth, 103), ''), 'No Date of Birth Found') AS individualDOB, 
-    ISNULL(NULLIF(individual.job_title, ''), 'No Occupation Found') AS individualOccupation,
+	CASE 
+	  WHEN individual.job_title IS NULL OR individual.job_title = ''
+		  THEN 'No Occupation Found'
+	  WHEN CHARINDEX('-', individual.job_title, 1) > 1
+		  THEN (SELECT trim(value) FROM STRING_SPLIT(individual.job_title, '-')  ORDER BY (SELECT NULL) OFFSET 1 ROWS FETCH NEXT 1 ROWS ONLY )
```
Comment (VasInsolvencyService) [inline INSS.EIIR.DataAccess/DatabaseScripts/getEiirIndex.sql:217]: why do we need FETCH NEXT 1 ROWS ONLY ?

File: INSS.EIIR.DataAccess/DatabaseScripts/getEiirExtractXml.sql (hunk 1 above)
Comment (prabhat-inss) [reply INSS.EIIR.DataAccess/DatabaseScripts/getEiirExtractXml.sql:283]: So what is your suggestion ?

File: INSS.EIIR.DataAccess/DatabaseScripts/getEiirIndex.sql (hunk 2 above)
Comment (prabhat-inss) [reply INSS.EIIR.DataAccess/DatabaseScripts/getEiirIndex.sql:217]: So what is your suggestion ?

File: INSS.EIIR.DataAccess/DatabaseScripts/getEiirExtractXml.sql (hunk 1 above)
Comment (VasInsolvencyService) [reply INSS.EIIR.DataAccess/DatabaseScripts/getEiirExtractXml.sql:283]: Can you check the outcome locally by removing FETCH NEXT 1 ROWS ONLY , if not required then remove it.

File: INSS.EIIR.DataAccess/DatabaseScripts/getEiirIndex.sql (hunk 2 above)
Comment (VasInsolvencyService) [reply INSS.EIIR.DataAccess/DatabaseScripts/getEiirIndex.sql:217]: Can you check the outcome locally by removing FETCH NEXT 1 ROWS ONLY , if not required then remove it.

**Review by VasInsolvencyService — APPROVED** [review]

**Pull request merged**